keyboards/pinky/3/3.c

```diff
@@ -0,0 +1,24 @@
+#include "pinky.h"
```

Comment:
Actually, please remove everything but this line. 

The rest of this is handled by core code and should not need to be called here. 

https://docs.qmk.fm/#/feature_audio